src/func_cache.c

```diff
@@ -445,7 +445,7 @@ ts_func_cache_get(Oid funcid)
 	return (NULL == entry) ? NULL : entry->funcinfo;
 }
 
-FuncInfo *
+TSDLLEXPORT FuncInfo *
```

Comment:
Actually it should not be any changes in this file :)
```suggestion
FuncInfo *
```